js/packages/openinference-vercel/src/utils.ts

```diff
@@ -368,12 +368,51 @@ const getOpenInferenceAttributes = (attributes: Attributes): Attributes => {
           };
         case AISemanticConventions.TOKEN_COUNT_COMPLETION:
         case AISemanticConventions.TOKEN_COUNT_PROMPT:
+          // Do not capture token counts for chain spans as they are captured by the LLM spans below
+          // Capturing on chains as well results in double token counts
+          if (spanKind === OpenInferenceSpanKind.CHAIN) {
+            return openInferenceAttributes;
+          }
+          return {
+            ...openInferenceAttributes,
+            [openInferenceKey]: attributes[convention],
+          };
         case AISemanticConventions.TOOL_CALL_NAME:
-        case AISemanticConventions.TOOL_CALL_ARGS:
           return {
             ...openInferenceAttributes,
             [openInferenceKey]: attributes[convention],
           };
+        case AISemanticConventions.TOOL_CALL_ARGS: {
```

Comment:
not sure what to put for `input.value` on tool spans so put args?

Reply:
Yeah they go into the input for now. We don't have any dedicated slot